src/timeout.rs

```diff
@@ -1,11 +1,27 @@
+//! Contains functions related to timeout of a process.
+
 use crate::error::SubmissionError;
 use std::{
-    process::{Child, ExitStatus, Output},
+    process::{ExitStatus, Output},
     time::Duration,
 };
-use tokio::time::{sleep, Instant};
+use tokio::{
```

Comment:
I wonder if you know, how they live in Tokyo

Reply:
I asked @andreaswurtz, and he told me